etc/openqa/workers.ini

```diff
@@ -83,6 +86,15 @@
 
 # The section ids are the instance of the workers.
 # The key/value pairs will appear in vars.json
+[1]
+WORKER_CLASS = qemu_x86_64_staging,qemu_x86_64,tap
+
+[2]
+WORKER_CLASS = qemu_x86_64_staging,qemu_x86_64,tap
+#PARALLEL_ONE_HOST_ONLY = 1
+[3]
+WORKER_CLASS = qemu_x86_64_staging,qemu_x86_64,tap
+
```

Comment:
no

Reply:
it is dumped PR. those are not meant to stay obviously

Reply:
I suggest you use https://open.qa/docs/#_customize_configuration_directory for tests like that. If you use my helper setup: It makes already use of this variable so editing this file here won't do anything for you anyway.